docs/advance-toolchain-documentation.html

```diff
@@ -347,11 +349,34 @@
 
 <p>YUM supports package upgrades for new revision releases (i.e. <i>5.0-0</i> to <i>5.0-1</i>). For new major releases (i.e. <i>5.0-8</i> to <i>6.0-1</i>), please proceed as in a normal installation.</p>
 <br>
+
+<h2><a name="powertools">Using IBM Linux on Power tools repository</a></h2>
+
+<p>Advance Toolchain is part of the IBM Linux on Power tools repository, you can get AT's repository configured by installing the ibm-power-repo package:</p>
+<ul>
+	<li>Download the RPM package from <a href="https://www14.software.ibm.com/webapp/set2/sas/f/lopdiags/yum.html">https://www14.software.ibm.com/webapp/set2/sas/f/lopdiags/yum.html</a>.</li>
+	<li>Install the ibm-power-repo RPM package, by running the command with root privileges:</li>
```

Comment:
s/the command/the following command/ sounds better, and consistent with later phrasing